server/models/data/Timeslot.js

```diff
@@ -0,0 +1,12 @@
+const UserTypes = require('../../constants/userTypes.json');
+
+class Timeslot {
+    // The password field will NOT be stored outside of the database for security purposes.
```

Comment:
Not sure if this comment is relevant for the specific code lines. It may be better to remove it to avoid confusion.

Reply:
Will do, good point